docs/src/languages/deno.md

```diff
@@ -4,8 +4,36 @@ Deno support is available through the [Deno extension](https://github.com/zed-in
 
 - Language server: [Deno Language Server](https://docs.deno.com/runtime/manual/advanced/language_server/overview/)
 
+## Deno Configuration
+
+To use the Deno Language Server with TypeScript and TSX files, you will likely wish to to disable the default language servers and enable deno by adding the following to your settings.json:
+
+```json
+{
+  "languages": {
+    "TypeScript": {
+      "language_servers": [
+        "deno",
+        "!typescript-language-server",
+        "!vtsls",
+        "!eslint"
+      ]
+    },
+    "TSX": {
+      "language_servers": [
+        "deno",
+        "!typescript-language-server",
+        "!vtsls",
+        "!eslint"
+      ]
+    }
+  }
+}
+```
+
+See [Configuring supported languages](https://zed.dev/docs/configuring-languages) in the Zed documentation for more information.
```

Comment:
Can we use a relative Markdown link here?

```suggestion
See [Configuring supported languages](../configuring-languages.md) in the Zed documentation for more information.
```

This way the link will still work when running the docs locally.